runtime/doc/develop.txt

```diff
@@ -173,10 +173,10 @@ Strict "vimdoc" subset:
   (soft-wrapped) instead of preformatted (hard-wrapped) layout common in
   legacy :help docs.
   - Limitation: currently the parser https://github.com/neovim/tree-sitter-vimdoc
-    does not understand numbered listitems, so use a bullet symbol (- or •)
+    does not understand numbered list items, so use a bullet symbol (- or •)
     before numbered items, e.g. "• 1." instead of "1.".
 - Separate blocks (paragraphs) of content by a blank line.
-- Do not use indentation in random places—that prevents the page from using
+- Do not use indentation in random places: that prevents the page from using
```

Comment:
this is an emdash, it's intentional. looks better on web: https://neovim.io/doc/user/develop.html#dev-doc